registry/remote/example_test.go

```diff
@@ -451,6 +479,56 @@ func ExampleRepository_Fetch_artifactReferenceManifest() {
 	// {"mediaType":"application/vnd.cncf.oras.artifact.manifest.v1+json","artifactType":"example/signature","blobs":null,"subject":{"mediaType":"application/vnd.oci.image.manifest.v1+json","digest":"sha256:00e5ffa7d914b4e6aa3f1a324f37df0625ccc400be333deea5ecaa199f9eff5b","size":24}}
 }
 
+// ExampleRepository_Fetch_blobsOfArtifactManifest gives an example of pulling the blobs
+// of an artifact manifest.
+func ExampleRepository_Fetch_blobsOfArtifactManifest() {
+	repo, err := remote.NewRepository(fmt.Sprintf("%s/%s", host, exampleRepositoryName))
+	if err != nil {
+		panic(err)
+	}
+	ctx := context.Background()
+
+	// fetch the artifact manifest described by exampleDescriptor
+	exampleDescriptor := ocispec.Descriptor{
+		MediaType: artifactspec.MediaTypeArtifactManifest,
+		Digest:    "sha256:73bccdadf23b5df306bf77b23e1b00944c7bbce44cf63439afe15507a73413b5",
+		Size:      398}
+	rc, err := repo.Fetch(ctx, exampleDescriptor)
+	if err != nil {
+		panic(err)
+	}
+	defer rc.Close()
+
+	// parse the pulled manifest and fetch its blobs
+	pulledContent, err := io.ReadAll(rc)
+	if err != nil {
+		panic(err)
+	}
+	fmt.Println(string(pulledContent))
+
+	var pulledManifest artifactspec.Manifest
+	if err := json.Unmarshal(pulledContent, &pulledManifest); err != nil {
+		panic(err)
+	}
+	for _, blob := range pulledManifest.Blobs {
+		rc, err := repo.Fetch(ctx, ocispec.Descriptor{
+			MediaType: blob.MediaType,
+			Digest:    blob.Digest,
+			Size:      blob.Size,
+		})
+		if err != nil {
+			panic(err)
+		}
+		defer rc.Close()
+		content, err := io.ReadAll(rc)
```

Comment:
We can leverage [content.FetchAll()](https://github.com/oras-project/oras-go/blob/main/content/storage.go#L59-L61).